### DISCUSSION TITLE
Running Airbyte on Docker Swarm (Portainer.io or Swarmpit)

### DISCUSSION THREAD

**Discussion by asijoumi** [airbytehq/airbyte#36969] [Connector Ideas and Features]
### Relevant information

Hi !

I would like to know how can I startup Airbyte on Docker Swarm without running a script shell.

For exemple, for Portainer.io or Swarmpit, we can only provide a docker-compose file without additional files.

Is there a solution for that ?

Thank you

**Comment by marcosmarxm** [comment]
Move this as a feature or idea discussion.
Anyone who want this please upvote the discussion.

**Comment by dpaes** [comment]
waiting for that too, cant find something consistent in docs, like this: https://stackoverflow.com/questions/75823212/deploying-airbyte-with-just-docker-compose-yaml-and-varibles

[a comment by an automated account is not shown]
